run_all_tests_unix.sh

```diff
@@ -9,12 +9,7 @@ tty -s && tput bold
 echo -n "Starting IPv8 testsuite: "
 if nosetests --version >>/dev/null 2>&1; then
     echo "using test runner 'nosetests'!"
-    # Use nosetests2 if available and if not, try version possibly incompatible with python2
-    if [ -x "$(command -v nosetests2)" ]; then
-        test_command="nosetests2"
-    else
-        test_command="nosetests"
-    fi
+    test_command="python run_nose.py"
```

Comment:
It's not safe to assume that the `python` command one ones system always points to a Python 2 interpreter. Maybe we should check specifically for `python2` availability first (like we did with `nosetests2`)?

Reply:
added a separate check for python interpreter version